sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/user/AnonymousUserRestControllerTest.java

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.domain.administration.user;
+
+import static com.mercedesbenz.sechub.test.RestDocPathParameter.ONE_TIME_TOKEN;
+import static com.mercedesbenz.sechub.test.SecHubTestURLBuilder.https;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.mercedesbenz.sechub.domain.administration.TestAdministrationSecurityConfiguration;
+import com.mercedesbenz.sechub.sharedkernel.Profiles;
+import com.mercedesbenz.sechub.spring.security.SecHubSecurityProperties;
+import com.mercedesbenz.sechub.test.TestPortProvider;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+@ContextConfiguration(classes = { AnonymousUserRestController.class })
+@ActiveProfiles({ Profiles.TEST })
+@Import(TestAdministrationSecurityConfiguration.class)
+public class AnonymousUserRestControllerTest {
```

Comment:
We have a naming convention for tests which do need spring boot container running (to identify the "potentially slow ones")

$xyzSpringBootTest

Please rename to
```suggestion
public class AnonymousUserRestControllerSpringBootTest {
```